src/main/java/fizzbuzz/Devider.java

```diff
@@ -0,0 +1,20 @@
+package fizzbuzz;
+
+public class Devider {
+    
+    private int firstDevider;
```

Comment:
what about if you want to have more dividers and combinations of them?
you need to add a new field and its getter every time